api/main.py

```diff
@@ -84,5 +87,6 @@ async def root_POST(
     changed = df.signature_type[df.signature_type.notna()]
     sig.loc[changed.index] = changed
     pred["signature_type"] = sig
+    pred = pred.dropna(subset=["signature_type"])
```

Comment:
When testing with `tyne_and_wear_hex` some of the data returned contained NaN's for signature type (same issue that we discussed over Slack).

Right now I've put in this line which matches what you did in `pipeline.py`, but more fundamentally, I'm guessing that this issue arises because the empty parquet contains tiles that we don't care about? So we might need to update those files.

Reply:
Even with the latest one? That should not happen.